core/controllers/topic_editor.py

```diff
@@ -418,13 +414,13 @@ def put(self, topic_id):
             'topic_and_subtopic_page_change_dicts')
         topic_and_subtopic_page_change_list = []
         for change in topic_and_subtopic_page_change_dicts:
-            if change['cmd'] == (
+            if change.cmd == (
                     subtopic_page_domain.CMD_UPDATE_SUBTOPIC_PAGE_PROPERTY):
                 topic_and_subtopic_page_change_list.append(
-                    subtopic_page_domain.SubtopicPageChange(change))
+                    subtopic_page_domain.SubtopicPageChange(change.to_dict()))
             else:
                 topic_and_subtopic_page_change_list.append(
-                    topic_domain.TopicChange(change))
+                    topic_domain.TopicChange(change.to_dict()))
```

Comment:
You don't need this, right? Since `change` is already a TopicChange object?

Reply:
Right, thanks!